src/routes/index.ts

```diff
@@ -22,4 +23,5 @@ export const routes = (app: Hono, konfigStore: Store<Record<string, unknown>>) =
   app.route("/latest_version", invidiousRouteLatestVersion);
   app.route("/api/manifest/dash/id", invidiousRouteDashManifest);
   app.route("/videoplayback", videoPlaybackProxy);
+  app.route("/health", health);
```

Comment:
If it's just a dumb 200 response then I would prefer to have /healthz
https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f